ShellPkg/Library/UefiShellDebug1CommandsLib/DmpStore.c

```diff
@@ -823,12 +823,6 @@ ShellCommandRunDmpStore (
       // Get the Name of the variable to find
       //
       Name = ShellCommandLineGetRawValue (Package, 1);
-      // MU_CHANGE [BEGIN] - CodeQL change

```

Comment:
This should break the CodeQL, how do we want to handle that?

Reply:
You can add it to here: https://github.com/microsoft/mu_basecore/blob/release/202208/CodeQlFilters.yml

Reply:
Please try to narrow it to the specific rule.

Reply:
Turned out that it was already in the exception list.